### PR TITLE
Improvements to coho list-pulls.

### PR DESCRIPTION
**Pull request by TimBarham**
1. Supports an environment variable (`CORDOVA_GIT_ACCOUNT`) to specify `<username>:<apitoken>` to avoid hitting GitHub rate limits.
2. Uses API to get most recent comment info, if available.

### CODE REVIEW

File: src/list-pulls.js [hunk 1]
```diff
@@ -23,7 +23,9 @@ var apputil = require('./apputil');
 var flagutil = require('./flagutil');
 var repoutil = require('./apputil');
 
-var GITHUB_API_URL = "https://api.github.com/";
+// Set env variable CORDOVA_GIT_ACCOUNT to <username>:<apitoken> to avoid hitting GitHub rate limits.
```
Comment (muratsu) [inline src/list-pulls.js]: my understanding was that it's just <apitoken> not <username>:<apitoken>
eg. 
curl -s -I -H Authorization: token <apitoken> https://api.github.com/users/apache/repos

Comment (TimBarham) [inline src/list-pulls.js]: Good point. Updated comment.

**Comment by muratsu** [comment]
LGTM :shipit:

**Comment by nikhilkh** [comment]
LGTM

**Pull request merged**